src/Components/Presentational/Chrome/Navigation.js

```diff
@@ -12,6 +12,8 @@ import RequestListItem from '../../Scenes/SideNavigation/RequestListItem/Request
 import KeysListItem from '../../Scenes/SideNavigation/KeysListItem/KeysListItem';
 import AwsAthenaListItem from '../../Scenes/SideNavigation/AwsAthenaListItem/AwsAthenaListItem';
 import AvatarListItem from '../../Scenes/SideNavigation/AvatarListItem/AvatarListItem';
+import RequestAssetsContainer from '../../Scenes/RequestAssets/RequestAsset/RequestAsset-Container';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```

Comment:
looks like (because the `<Router>`) is no longer used here, you can remove `BrowserRouter as Router` from that import block!

Reply:
(just a linting fix here)

Reply:
yeah I removed it. It was causing PR gate fail.